bedrock/landing/templates/landing/base-landing.html

```diff
@@ -0,0 +1,43 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "base-protocol.html" %}
```

Comment:
It's getting the default Mozilla favicon from the parent template. You can change that to the Firefox favicon:
```
{% block page_favicon %}{{ static('img/favicons/firefox/browser/favicon.ico') }}{% endblock %}
{% block page_favicon_large %}{{ static('img/favicons/firefox/browser/favicon-196x196.png') }}{% endblock %}
{% block page_ios_icon %}{{ static('img/favicons/firefox/browser/apple-touch-icon.png') }}{% endblock %}
```

Reply:
On second thought, you'll want to do this in each page template instead of in the shared base (I assume the VPN and Pocket pages will want their own favicons)

Reply:
Ah good call